docs/docs/channels/direct/slack.md

```diff
@@ -1,8 +1,9 @@
 # Slack
 
 On some channels as the Direct one, the user will need to add his provider integration credentials in order to provide Novu the right authorization to send the notification by his behave.
+<!-- @djabarovgeorge did you mean `on their behalf` ? sorry, not sure I understand.-->
```

Comment:
@oba2311 Could you please remove that comment here? We try not to leave comments in the code itself. You can use this PR to as a question for @djabarovgeorge and he will get a notification.

Reply:
Yes agree the text below will be more appropriate.
'On some channels such as the Direct one, the user will need to add his provider integration credentials in order to provide Novu with the right authorization to send the notifications on the user's behalf.'